### DISCUSSION TITLE
New plex server (same library) - start all over again?

### DISCUSSION THREAD

**Discussion by djnield** [meisnate12/Plex-Meta-Manager#81] [Q&A] [answered]
I've been successfully using version 1.1.0 of the script up to now.

Now, my plex server hardware has been changed so all the collection work I did has gone. I assumed I can just run the script again but I see v1.3.0 appears quite different in the config files.

What's the best way to proceed.

**Comment by meisnate12** [comment, marked as answer]
its up to but your old config should run just fine there hasn't been any breaking changes

**Reply by djnield** [reply]
I'd like to use your most recent config.

Will it begin again from zero (so a long time to complete) or somehow recognise I've already built up a cache?
Might the old cache interfere with the new config?

**Reply by meisnate12** [reply]
you can use the same cache file and it will interface with the new config file just fine

**Reply by djnield** [reply]
Perfect, let me try.

**Reply by djnield** [reply]
Made a rookie error but otherwise went smoothly.
Awesome talent you have!

**Reply by djnield** [reply]
<img width="1288" alt="Screenshot 2021-03-04 at 23 03 06" src="https://user-images.githubusercontent.com/53867992/110042728-19d89200-7d3e-11eb-9549-ae56c8cd1b90.png">
